packages/layout/src/composable/useLayout.ts

```diff
@@ -26,11 +97,11 @@ const PLUGIN_POSITION = {
   fixed: 'fixed'
 }
 
-const pluginState = reactive({
+const pluginState: IPluginState = reactive({
```

Comment:
```suggestion
const pluginState: IPluginState = reactive<IPluginState>({
```

vue的ref和reactive都支持范型，这样写可以更严格地校验数据结构
有多处类似代码建议更改，更改后可能会出现其他类型错误需要继续更改

Reply:
已调整